### PR TITLE
Add coins

### PR DESCRIPTION
**Pull request by tebanwolox**
## Summary

- Add create coin service

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,6 +67,7 @@
     }
   },
   "dependencies": {
+    "axios": "^0.19.0",
```
Comment (shernanm) [inline package.json]: Remove libraries you don't use

File: app/helpers/jwt.js [hunk 2]
```diff
@@ -4,3 +4,5 @@ const { secret, expirationToken } = require('../../config').common.session;
 
 exports.getToken = ({ id, firstName, lastName }) =>
   jwt.sign({ id, firstName, lastName }, secret, { expiresIn: expirationToken });
+
+exports.verifiedToken = token => jwt.verify(token, secret);
```
Comment (shernanm) [inline app/helpers/jwt.js]: To better control errors here, make this function asynchronous using a callback.

File: app/controllers/crypto_coin.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+exports.createCoin = (req, res, next) => {
+  logger.info('Start create coin');
+  const { user } = req;
+  const currency = req.body.currency.toUpperCase();
+  return verifiedCrypto(currency)
+    .then(response => {
+      if (response.success) return addCoin(currency, user.id);
+      throw errors.apiError('This is not a cryptocoin');
+    })
+    .then(coin => res.status(201).send({ coin }))
```
[the remark this answers is not in the document]
Comment (tebanwolox) [reply app/controllers/crypto_coin.js]: with this way, the response is returning.

File: app/services/breave_coin.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+const request = require('request-promise');
+
+const { braveCoinEnpoint, braveHost, braveApiKey } = require('../../config/index').common.braveCoin;
```
Comment (shernanm) [inline app/services/breave_coin.js]: You can skip going to the index file, this is done by default. For example `const { ... } = require('../config')`

**Pull request merged**